config/operator/Makefile

```
@@ -17,7 +17,7 @@ endif
 
 generate-all-in-one:
 	@ for yaml in $$(LC_COLLATE=C ls all-in-one/*yaml); do \
-		cat $$yaml && echo -e "\n---\n"; \
+		echo "---" && cat $$yaml ; \
```

I would tend to keep the additional safety `\n` and to use `printf` as @charith-elastic recommended:
```suggestion
		printf "\n---\n" && cat $$yaml ; \
```